credentials/src/test/kotlin/web5/sdk/credentials/StatusListCredentialTest.kt

```diff
@@ -50,12 +51,12 @@ class StatusListCredentialTest {
     val issuerDid = DidKey.create(keyManager)
     val holderDid = DidKey.create(keyManager)
 
-    val credentialStatus = StatusList2021Entry.builder()
-      .id(URI.create("cred-with-status-id"))
-      .statusPurpose("revocation")
-      .statusListIndex("123")
-      .statusListCredential(URI.create("https://example.com/credentials/status/3"))
-      .build()
+    val credentialStatus = BitstringStatusListEntry(
+      id = URI.create("cred-with-status-id"),
```

Comment:
for lots of did types we are writing, we opted to set id as a String. thoughts on using String instead of URI in this type's id field?

Reply:
@nitro-neal ^ ?

Reply:
its a URI type right? `URI.create("cred-with-status-id")`

I guess an offical one wouldn't look like this, just for testing

Reply:
ok, sounds good - i think i'm just violently against anything the spec says now because i'm mad at the specs